storagext/cli/src/cmd/market.rs

```diff
@@ -154,6 +157,20 @@ impl MarketCommand {
                     tracing::error!("Could not find deal parameters for {account_id}");
                 }
             }
+            MarketCommand::RetrieveAllDealParameters => {
+                let deal_parameters = client.retrieve_all_deal_parameters().await?;
+
+                match output_format {
+                    OutputFormat::Plain => {
+                        println!("[");
+                        for param in deal_parameters.iter() {
+                            println!("{}, ", param)
+                        }
+                        println!("]");
```

Comment:
```suggestion
                    OutputFormat::Plain => {
                        println!("{:#?}", deal_parameters);
```

I assume this didn't work here?